### ISSUE TITLE
Windows right click menu doesn't work after portable's folder changed

### ISSUE THREAD

**Issue by Sravdar** [gitextensions/gitextensions#11806]
### Environment

Windows 11
- Git Extensions 4.2.1.17611
- Build b0c0b284876b63c52c77195ada82a85bfadbb083
- Git 2.44.0.windows.1
- Microsoft Windows NT 10.0.22631.0
- .NET 6.0.25
- DPI 96dpi (no scaling)
- Portable: True
- Microsoft.WindowsDesktop.App Versions

```
    Microsoft.WindowsDesktop.App 6.0.25 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 8.0.6 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
```

### Issue description

When using portable, after changing the location of the folder right click menu for opening git extensions no longer works. Even when repaired and os restarted it did not worked. Icons reappeared when disabled and reenabled shell extensions but when git extensions is clicked on right click menu it doesn't respond.

### Steps to reproduce

After complete installation of portable git extensions move it to the another folder and try to use right click context menu to lauch it.

**Comment by pmiossec** [comment]
I tend to think that it is fixed by "fix(ShellEx): Write InstallDir to registry on registration of the shell extension" of #11770 

Could you test development portable version? That you will find here: https://github.com/gitextensions/gitextensions?tab=readme-ov-file#downloads

And confirm if it fixes the issue...

**Comment by pmiossec** [comment]
> Yes, re-enabling shell extensions fixed the issue on 4.3.0.17862.

Thanks.
